admin/static/js/reql_docs.js

```diff
@@ -31,8 +31,8 @@ reql_docs = {
     "url": "add"
   }, 
   "api/javascript/and/": {
-    "body": "bool.and(bool[, bool, ...]) &rarr; boolr.and(bool[, bool, ...]) &rarr; bool", 
```

Comment:
This file is auto generated. These changes should be made in the corresponding files in https://github.com/rethinkdb/docs/tree/master/api